extensions/notebook/src/jupyter/jupyterServerInstallation.ts

```diff
@@ -55,6 +57,52 @@ export class JupyterServerInstallation {
 
 	public static readonly DefaultPythonLocation = path.join(utils.getUserHome(), 'azuredatastudio-python');
 
+	private _prompter: IPrompter;
+	private readonly _expectedPythonPackages: PythonPkgDetails[] = [
+		{
+			name: 'jupyter',
+			version: '1.0.0'
+		}, {
+			name: 'pandas',
+			version: '0.24.2'
```

Comment:
we hardcode package versions here in a number of places now. Can we at least save package versions in some constants somewhere

Reply:
Consolidated these into some common lists.